src/amo/reducers/collections.js

```diff
@@ -73,6 +76,9 @@ export type CollectionsState = {
       |};
     },
   },
+  collectionUpdates: {
+    [collectionSlug: string]: {| updating: boolean, success?: boolean |},
```

Comment:
`successful` might be a slightly better flag name and might make the code read a bit nicer, as I'm assuming this flags that an update was successful.